action.yml

```diff
@@ -54,6 +54,11 @@ runs:
               message: ${{ github.event.client_payload.commitMessage }}
               skip-empty: true
 
+        - name: Store changed files
```

Comment:
```suggestion
        - name: Store number of changed files
```

Wonder if we should also rename the payload parameter?

Reply:
Sure, what about `changedFileCount` or `numberOfChangedFiles` then?

Reply:
Either works!

Reply:
numberOfChangedFiles